aopy/preproc/wrappers.py

```diff
@@ -84,6 +86,32 @@ def proc_single(data_dir, files, preproc_dir, subject, te_id, date, preproc_jobs
         broadband_data, broadband_metadata = aodata.load_preproc_broadband_data(preproc_dir_base, subject, te_id, date)
         assert broadband_data.shape == (broadband_metadata['n_samples'], broadband_metadata['n_channels'])
         files['broadband'] = broadband_filename # for proc_lfp()
+    if 'spikes' in preproc_jobs:
```

Comment:
is it possible to move `spikes` to the end? for ecube data we will want to make sure to process ap band before processing spikes